pimd/subdir.am

```diff
@@ -89,6 +89,7 @@ pimd_pim6d_SOURCES = \
 	pimd/pim6_main.c \
 	pimd/pim6_stubs.c \
 	pimd/pim6_cmd.c \
+	pimd/pim_register.c \
```

Comment:
The register changes don't belong in the same commit with the mroute changes. Please split.

Reply:
Sure will split